CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## UNRELEASED
 
+## 1.1.0-beta1 (October 4, 2022)
+
```

Comment:
```suggestion
```

Reply:
Yeah, that's a byproduct of running the script multiple times, this wouldn't be a nominal scenario.

I'm going to drop the example commits before completing the PR